lurk-fibonacci/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "lurk-fibonacci"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+camino = { workspace = true }
+lurk = { path = "../", package = "lurk" }
+once_cell = { workspace = true }
+pasta_curves = { git = "https://github.com/lurk-lab/pasta_curves", branch = "dev" }
+tracing = { workspace = true }
+
+[dev-dependencies]
+criterion = "0.5"
+cfg-if = "1.0.0"
+
+[build-dependencies]
+vergen = { version = "8", features = ["build", "git", "gitcl"] }
```

Comment:
These are copied from `.Cargo.toml`. Is there a way to avoid copying?

Reply:
No